Stripe/STPAPIClient.m

```diff
@@ -847,14 +846,13 @@ - (void)createPaymentMethodWithParams:(STPPaymentMethodParams *)paymentMethodPar
                                  completion:(STPPaymentMethodCompletionBlock)completion {
     NSCAssert(paymentMethodParams != nil, @"'paymentMethodParams' is required to create a PaymentMethod");
     NSCAssert(paymentMethodParams.rawTypeString != nil, @"Set the `type` or `rawTypeString` property on paymentMethodParams.");
+    [[STPAnalyticsClient sharedClient] logPaymentMethodCreationAttemptWithConfiguration:self.configuration paymentMethodType:paymentMethodParams.rawTypeString];
+    
     [STPAPIRequest<STPPaymentMethod *> postWithAPIClient:self
                                                endpoint:APIEndpointPaymentMethods
                                              parameters:[STPFormEncoder dictionaryForObject:paymentMethodParams]
                                            deserializer:[STPPaymentMethod new]
                                              completion:^(STPPaymentMethod *paymentMethod, __unused NSHTTPURLResponse *response, NSError *error) {
-                                                 if (error == nil && paymentMethod != nil) {
```

Comment:
I don't know why this didn't match the other analytics, which fire regardless of success.

Reply:
@csabol-stripe Can you confirm I'm not clobbering something intentional here?